### DISCUSSION TITLE
Underline option

### DISCUSSION THREAD

**Discussion by ramithuh** [ahrm/sioyek#615] [Q&A] [answered]
Hi, I'm new to sioyek, I was wondering whether I can configure an underline option manually.

Something like below.

TIA!

<img width="405" alt="image" src="https://user-images.githubusercontent.com/3503098/215337412-73c1fd1f-19e4-43d9-b0c1-aa2d965e9561.png">

<img width="697" alt="image" src="https://user-images.githubusercontent.com/3503098/215337427-bdc31f54-86ae-4638-8024-f823690e5ab8.png">

**Comment by ahrm** [comment, marked as answer]
This is not currently possible.
